tests/components/home_connect/test_time.py

```diff
@@ -134,13 +135,14 @@ async def test_time_entity_error(
     with pytest.raises(HomeConnectError):
         getattr(problematic_appliance, mock_attr)()
 
-    await hass.services.async_call(
-        TIME_DOMAIN,
-        SERVICE_SET_VALUE,
-        {
-            ATTR_ENTITY_ID: entity_id,
-            ATTR_TIME: time(minute=1),
-        },
-        blocking=True,
-    )
+    with pytest.raises(ServiceValidationError):
```

Comment:
```suggestion
    with pytest.raises(ServiceValidationError, match=".."):
```
Can be used to test if the right error is raised